### ISSUE TITLE
Diamond Theme: topbar menu button is not activated by ENTER key

### ISSUE THREAD

**Issue by miedrl** [primefaces/primefaces#8065]
**Describe the defect**
To fulfil accessibility success criterion [2.1.1](https://www.w3.org/TR/WCAG21/#keyboard-accessible) "[a]ll functionality of the content is operable through a keyboard interface[...]". In Diamond Theme it is not possible to hide/show navigation by pressing ENTER on the topbar menu button, so this button is not fully accessible by keayboard.

**Environment:**
 - PF Version: _10.0_
 - Theme: Diamond

**To Reproduce**
1. Go to [Diamond Showcase](https://www.primefaces.org/diamond/)
2. Use tab key to focus navigation toggle button within topbar.
3. Try to trigger the button's action by pressing ENTER key.
4. See error > button's action is not triggered i.e. the navigation is not collapsed/expanded.

**Expected behavior**
For accessibility the navigation should be toggleable by pressing ENTER on its toggle button.

**Comment by jepsar** [comment]
That is not a PrimeFaces component. Please report theme issues at https://forum.primefaces.org/viewforum.php?f=136

**Comment by miedrl** [comment]
Okay, thank you.
